cmd/gitops/ui/run/cmd.go

```diff
@@ -80,9 +109,47 @@ func runCmd(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not create kube http client: %w", err)
 	}
 
-	profilesConfig := server.NewProfilesConfig(rawClient, helmRepoNamespace, helmRepoName)
+	profilesConfig := server.NewProfilesConfig(rawClient, options.HelmRepoNamespace, options.HelmRepoName)
+
+	var authConfig *auth.AuthConfig
+
+	if server.AuthEnabled() {
+		_, err := url.Parse(options.OIDC.IssuerURL)
+		if err != nil {
+			return fmt.Errorf("invalid issuer URL: %w", err)
+		}
+
+		redirectURL, err := url.Parse(options.OIDC.RedirectURL)
+		if err != nil {
+			return fmt.Errorf("invalid redirect URL: %w", err)
+		}
+
+		var oidcIssueSecureCookies bool
+		if redirectURL.Scheme == "https" {
+			oidcIssueSecureCookies = true
+		}
+
+		oidcCookieDuration, err := time.ParseDuration(options.OIDC.CookieDuration)
```

Comment:
You could simplify this by using https://pkg.go.dev/github.com/spf13/pflag#DurationVar above